### PR TITLE
Remove _.includes from isCurrentUserMaybeInGdprZone

### PR DESCRIPTION
**Pull request by jsnajdr**
A little Lodash removal I did while reviewing the cookie banner changes in #59848. Removes `_.includes` use.

### CODE REVIEW

**Comment by sgomes** [comment]
Thank you, @jsnajdr! I already had the lodash removal part of this improvement as part of #59848, together with a few other bits. Should I review this separately and rebase my PR on top of it, or should we review #59848 as a whole instead?

File: client/lib/analytics/utils/is-current-user-maybe-in-gdpr-zone.js
Comment (tyxla) [inline client/lib/analytics/utils/is-current-user-maybe-in-gdpr-zone.js]: It's nice we're no longer re-declaring it on every call 👍

**Review by tyxla — APPROVED** [review]
LGTM 👍 

Feel free to merge independently or as part of #59848

**Review by sgomes — APPROVED** [review]
Given you already have an approval from @tyxla, I'll go ahead and give you another one 🙂

Once this is merged, I'll rebase and modify the changes in my PR to match yours 👍

**Comment by jsnajdr** [comment]
@sgomes I'm sorry, I didn't notice that change in your patch. I was probably looking at at the cookie-related changes and ignored the irrelevant parts of the diff.

I'll merge this separately, after pushing another commit that does the TS conversion.

**Comment by sgomes** [comment]
> @sgomes I'm sorry, I didn't notice that change in your patch. I was probably looking at at the cookie-related changes and ignored the irrelevant parts of the diff.

@jsnajdr No worries, glad we thought of doing the same 🙂 My PR is now rebased on your changes.

**Pull request merged**
